test/lib/models/account_summary.js

```diff
@@ -25,6 +55,11 @@ describe('Symbol Details model', () => {
   testModelValidation({
```

Comment:
`describe('Symbol Details model', () => {` can we correct also the test description here, I see that I made a mistake here a while ago :smile: